extensions/2.0/Khronos/KHR_node_selectability/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Extending Nodes
+
+The `KHR_node_selectability` extension object is added to the objects within the `nodes` array. The extension object contains a single boolean `selectable` property. This value is mutable through JSON pointers as defined in the glTF 2.0 Asset Object Model and controls “selectability” of the node that contains it and all its children nodes recursively. A value of `false` causes all nodes below in the hierarchy to be omitted from “selection”, even any nodes below that have a value of `true`.
+
+|                | Type      | Description                               | Required            |
+|----------------|-----------|-------------------------------------------|---------------------|
+| **selectable** | `boolean` | Specifies whether the node is selectable. | No, default: `true` |
+
+In other words, a node is “selectable” if and only if there exists no node including or above it in the hierarchy where such node has a `selectable` property with value `false`. This allows a single change of a `selectable` property at a high level of the hierarchy to enable or disable the selection of complex (multi-node) objects.
+
+In the following example, both nodes (and therefore their meshes) are initially unselectable.
+
+```json
+{
+    "nodes": [
+        {
+            "children": [1],
+            "mesh": 0,
+            "extensions": {
+                "KHR_node_selectability": {
+                    "selectable": false
+                }
+            },
+        },
+        {
+            "mesh": 1
+        }
+    ]
+}
+```
+
+## Extending glTF 2.0 Asset Object Model
+
+The following pointer template represents the mutable property defined by this extension.
+
+| Pointer                                                  | Type   |
+|----------------------------------------------------------|--------|
+| `/nodes/{}/extensions/KHR_node_selectability/selectable` | `bool` |
+
+## Interaction with KHR_interactivity
+
+Unless used in conjunction with `KHR_interactivity`, the result of a “select” action on a given object is left up to the implementation.
+
+When used in conjunction with `KHR_interactivity`, a new interactivity event is defined as follows:
+
+|                         |                  | |
+|-------------------------|------------------|-|
+| **Type**                | `event/onSelect` | Select event |
+| **Configuration**       | `int nodeIndex`        | Index of a node |
+|                         | `bool stopPropagation` | Whether to allow parents of this node to also receive the event |
+| **Output values**       | `int selectedNodeIndex`  | Index of the actual node that was selected |
+|                         | `float3 selectionPoint`   | Position of intersection of the select ray with the geometry of the selected node in global space |
+|                         | `float3 selectionRayOrigin` | Position of the origin of the select ray in global space |
```

Comment:
In an earlier section, this was defined as the "selection ray", not the "select ray".

```suggestion
|                         | `float3 selectionPoint`   | Position of intersection of the selection ray with the geometry of the selected node in global space |
|                         | `float3 selectionRayOrigin` | Position of the origin of the selection ray in global space |
```

Reply:
Good catch!